src/anm/player.js

```diff
@@ -375,20 +375,22 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
                     if (player.anim === result) { // avoid race condition when there were two requests
                         // to load different animations and first one finished loading
                         // after the second one
-                        player.state.happens = C.LOADING;
-                        player.fire(C.S_CHANGE_STATE, C.LOADING);
-                        player.fire(C.S_LOAD, result);
-                        player._updateMediaVolumes();
-                        if (!player.handleEvents) player.stop();
-                        player._callPostpones();
-                        if (callback) callback.call(player, result);
-                        // player may appear already playing something if autoPlay or a similar time-jump
-                        // flag was set from some different source of options (async, for example),
-                        // then the rule (for the moment) is: last one wins
-                        if (player.autoPlay) {
-                            if (player.state.happens === C.PLAYING) player.stop();
-                            player.play();
-                        }
+                        utils.postpone(function(){
```

Comment:
Actually there are already some postponing mechanics which delay the calls to `play`/`pause`/`stop`/... methods util the resources will finish loading (no matter, from cache or not), i.e. [here](https://github.com/Animatron/player/blob/e0ed315637c317bdbf7493430eb0250ea3879a7c/src/anm/player.js#L569). Following the description, I suppose they are somehow failing in this case. I could be wrong, though—have you considered these methods to fail or it's the different problem?